codes/quantum/qubits/quantum_cyclic.yml

```diff
@@ -0,0 +1,117 @@
+#    \cite{manual:{(enter citation line incl. author and year here)}}
+# External websites such as code tables, coding theory packages, github pages linked as
+#    \url{https://example.com/example}
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are 
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: quantum_cyclic
+physical: qubits
+logical: qubits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title is singular
+name: 'Quantum-Cyclic code'
+
+# First reference with the code (optional).
+introduced: '\cite{arXiv:1007.1697}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'Every linear stabilizer code \(C\) has an associated totally isotropic subspace \(S\subset F_2^{2n}\). Quantum cyclic codes are those codes where \(S\) and its centralizer are ideals in the polynomial ring \(F_4[X]/X^n-1\).'
+
+# protection: 'Protects against ... Pauli noise. Approximate code with parameters ... for noise model ... .'
+
+# This field starts a list of specific labeled subfields; do not leave it
+# empty. If empty, comment out. Also, indentations are important!
+features:
+
+  # The fields below relate to specific properties of the code that are worth
+  # noting. In some cases (see below), there might be multiple pieces of
+  # information associated with a field (e.g., mutliple encoders/decoders,
+  # multiple facts about fault tolerance, etc.); in such cases, the field
+  # accepts a list of text entries, with one item per piece of information.
+  #
+  # Any fields below should be omitted if they do not apply.
+
+  rate: 'Unknown.'
+
+  # This field that starts a list of strings, indexed by a dash; do not leave it
+  # empty. If empty, comment out.
+  # encoders: 
+  #   - 'Unitary circuit of depth ... \cite{arxiv:old-paper}.'
+  #   - 'Measurement-based preparation ... with ancilla overhead of ... .'
+  #   - 'Leave discussion of fault tolerance to fault-tolerance field.'
+
+  # transversal_gates: 'Transversal ... gates \cite{doi:ok-paper}. Comment out if doesn''t apply.'
+
+  # general_gates:
+  #   - 'Universal gate set achieved by either additional ... gate.'
+  #   - 'Magic-state distillation protocols'
+  #   - 'kth Clifford hierarchy gates obtained by ... circuits'
+
+  decoders: 
+    - 'Adapted from the Berlekamp decoding algorithm for classical BCH codes \cite{arXiv:1007.1697}.'
+    # - 'Just-in-time decoder with ... \cite{arxiv:awesome-paper}.'
+
+  # fault_tolerance:
+  #   - 'Transversal gates are fault-tolerant w.r.t. ... noise \cite{doi:ok-paper}'
+  #   - 'Other fault-tolerant gadgets (measurements, encoders, error correcting steps)'
+  #   - 'Noise-model-preserving gadgets, noise-biased gates, fault-tolerant flag error correction'
+  #   - 'Pieceable fault tolerance.'
+
+  threshold: 
+    - 'Unknown'
```

Comment:
You do this above as well. I'm not convinced it's useful to say "unknown" for rate and threshold; I would just comment out this field.